scripts/uptimerobot.js

```diff
@@ -0,0 +1,36 @@
+// Description:
+//    Send a message from a UptimeRobot webhook
+//    You need to add the room attribute to your hook (for example
+//    ops@muc.example.com)
+//
+//     Example UptimeRobot webhook (the ampersand is required):
+//     https://example.com/incoming/uptimerobot/secret?room=ops@example.com&
+//
+//     Example request:
+//     https://example.com/incoming/uptimerobot/secret?room=ops@example.com&monitorFriendlyName=Main%20site&alertTypeFriendlyName=Up&monitorID=23779464475
+//
+// Configuration:
+//   WEBHOOK_TOKEN: A string for building your secret webhook URL
+//
+// Authors:
+//   Greg Karékinian <greg@5apps.com>
+
+(function () {
+  "use strict";
+
+  module.exports = function(robot) {
+    robot.router.get('/incoming/uptimerobot/'+ process.env.WEBHOOK_TOKEN, (request, response) => {
+      var html_decode = require("ent/decode");
```

Comment:
Requiring external code is usually done outside of any exported functions. In this case, it would be loaded on every request.